### ISSUE TITLE
Error: Action execution failed: AMP-ANIMATION start: 'e' is not a function

### ISSUE THREAD

**Issue by zhouyx** [ampproject/amphtml#21118]
Error: Action execution failed: AMP-ANIMATION start: 'e' is not a function
at (action-execution-failed-amp-animation-start-e-is-not-a-function.js:1)

Pretty low volume
go/amp-error/COWJ0_z8w8K8NA

**Comment by wassgha** [comment]
Don't see it in the logs anymore. Could you please confirm that this is gone?

**Comment by wassgha** [comment]
Closing for now. If this resurfaces again please reopen.
